force-app/main/triggers/classes/HOT_WorkOrderHandler.cls

```diff
@@ -42,6 +42,21 @@ public without sharing class HOT_WorkOrderHandler extends MyTriggers {
         for (WorkOrder workOrder : (List<WorkOrder>) records) {
             requestIds.add(workOrder.HOT_Request__c);
         }
+       
+        CustomNotificationType notificationType = [
+            SELECT Id, DeveloperName
+            FROM CustomNotificationType
+            WHERE DeveloperName = 'HOT_NotifyUser'
+        ];
+        for (WorkOrder workOrder : (List<WorkOrder>) records) {
+            Set<String> recipients = new Set<String>();
+            if(WorkOrder.HOT_Request__r.type__c=='User'){
```

Comment:
Stor T i type.

Er det mulig å benytte for-loopen over?

Reply:
Kunne du sett på om det er riktig nå, og hvorfor i huleste det ikke testen går innom den if setningen?